net/lighttpd-https-cert/files/ssl-enable.conf

```diff
@@ -6,8 +6,7 @@
 
 ssl.engine = "enable"
 ssl.pemfile = "/etc/lighttpd-self-signed.pem"
-ssl.use-sslv2 = "disable"
-ssl.use-sslv3 = "disable"
+ssl.openssl.ssl-conf-cmd = ("Protocol" => "-ALL, +TLSv1.2")
```

Comment:
I agree with TLS 1.2 only but we would include this patch for some future "major" release (like 3.12 and  4.1)